src/main/java/com/webmuffins/rtsx/board/constants/HTTPConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.webmuffins.rtsx.board.constants;
+
+public class HTTPConstants {
```

Comment:
Is it possible to rewrite it to Enum?

Reply:
Here will be string constants for HTTP, e.g. headers. And it is not a enumeration, like a permissions because they are not bring any business logic. It`s just a string patterns, that can be reused. So it is a good practice to make constant classes